src/renderer/views/Playlist/Playlist.js

```diff
@@ -80,6 +80,10 @@ export default Vue.extend({
           channelId: result.author ? result.author.channelID : '',
           infoSource: 'local'
         }
+        const channelThumbnailUrl = this.infoData.channelThumbnail.replace('=s48', '=s176')
```

Comment:
I totally can't understand `replace('=s48', '=s176')` part
Can you add code comment to explain it

Same for other places with same code

Reply:
the s is just the thumbnail size, i think